compositor_pipeline/src/pipeline/webrtc/whip_input/state.rs

```diff
@@ -14,7 +15,62 @@ use crate::pipeline::webrtc::{
 };
 
 #[derive(Debug, Clone, Default)]
-pub(crate) struct WhipInputsState(Arc<Mutex<HashMap<InputId, WhipInputConnectionState>>>);
+pub(crate) struct WhipInputsState(Arc<Mutex<WhipInputsStateInner>>);
+
+#[derive(Debug, Clone, Default)]
+struct WhipInputsStateInner {
+    /// Maps IDs without prefix to ones with prefix
+    prefixed_input_ids: HashMap<InputId, InputId>,
```

Comment:
You already have a hash map based on id, there is no point in having 2 independent maps that needs to be synce together